src/Test/Diagnostics/TestingOnly_WaitingService.cs

```diff
@@ -1,29 +1,25 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
-using System.Collections.Generic;
 using System.Composition;
-using System.Linq;
-using System.Threading.Tasks;
+using System.Windows.Threading;
 using Microsoft.CodeAnalysis.Shared.TestHooks;
-using Roslyn.Test.Utilities;
+using Roslyn.Utilities;
 
 namespace Roslyn.Hosting.Diagnostics.Waiters
 {
     [Export, Shared]
     public class TestingOnly_WaitingService
```

Comment:
Do we even need this type anymore? It seems like the other type does all the same stuff?

Reply:
this is public. this is for ones that doesn't have access to roslyn, but want to use waiter as long as they can somehow get to this Roslyn.xxx.Disagnostics.dll

Reply:
also, our integration test uses it. so.. I left it.